src/renderer/store/handlers/users.ts

```diff
@@ -95,18 +115,19 @@ export const ReceivedUser = (values) => {
 };
 
 export interface IUser {
```

Comment:
Since you've created `User` class, I don't think you also need `IUser` interface. Wherever you're using `IUser` you can change to `User`